iaac/terraform/aws-infra/s3/main.tf

```diff
@@ -0,0 +1,17 @@
+resource "aws_s3_bucket" "artifact_store" {
```

Comment:
shall we apply the https only policy by default?

Reply:
will add a todo for now, dont want to test new configurations so i can focus on the cognito and cognito-rds-s3 PRs